met-web/src/components/common/Modals/types.tsx

```diff
@@ -3,4 +3,5 @@ export interface NotificationModalProps {
     subText: string[];
     handleConfirm?: () => void;
     handleClose?: () => void;
+    tos?: string;
```

Comment:
what's tos

Reply:
is there a better name?

Reply:
terms of service, we could call it block text

Reply:
termsOfService maybe